src/pass/shape_inference_pass.cpp

```diff
@@ -36,24 +36,16 @@ class ShapeInferencePass : public mlir::FunctionPass<ShapeInferencePass> {
 
     // Populate the worklist with the operations that need shape inference:
     // these are operations that return a dynamic shape.
-    llvm::SmallPtrSet<mlir::Operation *, 16> op_worklist;
+    llvm::SmallVector<mlir::Operation *, 16> op_worklist;
```

Comment:
What's the reason for changing the container type from Set to a Vector? The change in container semantics here is quite significant.

Reply:
SmallPtrSet can't hold more than 32 elements.